lib/jsonapi/query_builder/mixins/sort.rb

```diff
@@ -43,8 +49,15 @@ def default_sort(options)
           # @param [Symbol] attribute The "sortable" attribute
           # @param [proc, Class] sort A proc or a sort class, defaults to a simple order(attribute => direction)
           def sorts_by(attribute, sort = nil)
-            sort ||= ->(collection, direction) { collection.order(attribute => direction) }
-            @supported_sorts = {**supported_sorts, attribute => sort}
+            @supported_sorts = {**supported_sorts, attribute => Sort::Static.new(attribute, sort)}
+          end
+
+          # Registers attribute that can be dynamically used for sorting. Sorting parameters are usually parsed from the
+          # `json:api` sort query parameter in the order they are given.
+          # @param [Symbol] attribute_prefix The "sortable" attribute prefix, e.g. `:data` for sorting by `data.name` and `data.created_at`
+          # @param [proc, Class] sort A proc or a sort class, defaults to a simple order(attribute => direction)
```

Comment:
Is this comment valid for this method?

Reply:
It isn't, copy paste leftover. I'll come back and do the docs (Readme too) in a later commit